src/app/helptext/sharing/smb/smb.ts

```diff
@@ -14,6 +15,11 @@ export const helptext_sharing_smb = {
     placeholder_name: T('Name'),
     tooltip_name: T('Enter a name for the share.'),
 
+    placeholder_comment: T("Comment"),
```

Comment:
As above.

Reply:
See my last commit, I have renamed Comment description in SMB as well as AFP, NFS and WebDav (will make tickets for the other places this should probably be done in)